### DISCUSSION TITLE
Looking for an API key to build a discord bot.

### DISCUSSION THREAD

**Discussion by Aeval** [massif-press/compcon#2118] [Archive]
Hey all,

Basically the title here. I've checked out WitchDice and FoundryVTT, since both utilize share codes to pull Pilot data, but they do so from api.compcon.app using a secret API key. I know the UNCLE bot exists, but it lacks the share code functionality and a few other features I was looking to build out for my server. I was hoping someone here could help me acquire an API key for my bot to use to api.compcon.app.

Thanks!

**Comment by jarena3** [comment]
sorry! forgot to check on this discussion. Just want to confirm that you've got what you need now
